src/client/sandbox/node/window.js

```diff
@@ -525,6 +532,30 @@ export default class WindowSandbox extends SandboxBase {
         nativeMethods.objectDefineProperty
             .call(window.Object, window.MessageEvent.prototype, 'origin', originPropDescriptor);
 
+        if (window.PerformanceEntry) {
+            if (nativeMethods.performanceEntryNameGetter) {
+                const namePropDescriptor = nativeMethods.objectGetOwnPropertyDescriptor
+                    .call(window.Object, window.PerformanceEntry.prototype, 'name');
+
+                namePropDescriptor.get = function () {
+                    const entry = this;
+                    const url   = nativeMethods.performanceEntryNameGetter.call(this);
+
+                    if (isPerformanceNavigationTiming(entry)) {
```

Comment:
Add separate tests for `isPerformanceNavigationTiming` function.
For example see this https://github.com/DevExpress/testcafe-hammerhead/blob/6071663c7f72afe6bd74e72d6ab3b3628f9ce1ab/test/client/fixtures/utils/dom-test.js#L167